src/colors.jsx

```diff
@@ -1,10 +1,15 @@
 import React from 'react';
 
-const Colors = (props) =>{
+// const Colors = (props) =>{
+const Colors = ({colorObj , toggle}) =>{
+    
+    // Below I took that colorObj from props that alreadys holds what you need from when you passed it through in App,js
+    const { color, lock } = colorObj; 
     return (
-        <div style={{backgroundColor: props.color}} className="w-100 d-flex flex-column align-items-center justify-content-center">
-            <h1>{props.hex}</h1>
-            <button className="btn btn-dark" onClick={props.toggle}>{props.lock ? 'UNLOCK': 'LOCK'}</button>
+        <div style={{backgroundColor: color}} className="w-100 d-flex flex-column align-items-center justify-content-center">
+            {/* <h1>{props.hex}</h1> */}
+            <h1>{ color }</h1>
+            <button onClick={toggle} className={lock ? 'btn btn-dark' : 'btn btn-outline-dark'}>{lock ? 'UNLOCK': 'LOCK'}</button>
         </div>
     )
 }
```

Comment:
In your color component I did some destructuring . I'll leave this to you to try and understand what I did as a learning point. Let me know if it doesn't make sense.